src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
Could this be simpler? It seems a bit wasteful to loop through all the observations when the same information is already computed. The deviance is almost the loglikelihood, `deviance(y,mu) = 2loglikelihood(y,y) - 2loglikelihood(mu,y)`, so we could define `loglikelihood(f) = -deviance(f)/2` since it only deviates from the joint logpdf with an additive constant.